src/WebServerManager.cpp

```
@@ -1821,7 +1831,7 @@ void WebServerManager::handleApiWriteTigerTag() {
 void WebServerManager::handleApiWriteOpenTag3D() {
     Serial.println("WebServerManager: POST /api/write-opentag3d received");
 
-    StaticJsonDocument<512> doc;
+    JsonDocument doc;
```

Clang (14.0.6)

[warning] variable 'doc' is not initialized

(cppcoreguidelines-init-variables)